nodes/nodes/eco-node-maintenance-operator.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: ASSEMBLY
+[id="node-maintenance-operator"]
+= Using the Node Maintenance Operator to place nodes in maintenance mode
+include::modules/common-attributes.adoc[]
+:context: node-maintenance-operator
+toc::[]
+
+You can use the Node Maintenance Operator to place nodes under maintenance mode. This is a standalone version of the Node Maintenance Operator that is independent of OpenShift Virtualization installation.
+
+[NOTE]
+====
+If you have installed OpenShift Virtualization, you must use the Node Maintenance Operator that is bundled with it. 
+====
+
+include::modules/eco-about-node-maintenance-standalone.adoc[leveloffset=+1]
+
+include::modules/eco-maintaining-bare-metal-nodes.adoc[leveloffset=+1]
+
+You can install the Node Maintenance Operator using the web console or the OpenShift CLI (`oc`).
+
+
+Place a node into maintenance from the CLI or using a `NodeMaintenance` custom resource.
```

Comment:
Maybe combine/move this line to the last line (installation).

Reply:
@razo7 Both are different procedures so can't combine them. With the addition of installation sections, it will work fine